verifysignature.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+# make a temporary directory
+rand_dir=$(openssl rand -base64 12)
+temp_dir="/tmp/$rand_dir"
+mkdir -p "$temp_dir"
+
+# ensure temp directory gets deleted on exit
+trap "rm -rf $temp_dir" EXIT
+
+config_file=$1
+key_file=$2
+
+# grab out the relevant parts from the config file
+signature=$(cat $config_file | jq -r .signature)
+message=$(cat $config_file | jq .config | jq -r '@json')
```

Comment:
Is this going to be run on CI? If so, are we sure that `jq` will be available?

Reply:
this is more for checking it locally so don't think should be an issue - I'll add a note to the README though that `jq` needs to be installed, good shout